airflow/ui/rules/unicorn.js

```diff
@@ -654,25 +654,6 @@ export const unicornRules =
        */
       [`${unicornNamespace}/prefer-array-find`]: ERROR,
 
-      /**
-       * Prefer `Array#flatMap` over `Array#map(…).flat()`.
-       *
-       * @example
-       * ```typescript
-       * // ❌ Incorrect
-       * const foo = bar.map(unicorn).flat();
-       * const foo = bar.map(unicorn).flat(1);
-       *
-       * // ✅ Correct
-       * const foo = bar.flatMap(unicorn);
-       * const foo = bar.map(unicorn).flat(2);
-       * const foo = bar.map(unicorn).foo().flat();
-       * const foo = bar.flat().map(unicorn);
-       * ```
-       * @see [unicorn/prefer-array-flat-map](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-array-flat-map.md)
-       */
-      [`${unicornNamespace}/prefer-array-flat-map`]: ERROR,
```

Comment:
This is an example of "natural" sorting. `prefer-array-flat` is now before `prefer-array-flat-map`